go.mod

```diff
@@ -419,7 +419,16 @@ require (
 
 require google.golang.org/grpc/examples v0.0.0-20221020162917-9127159caf5a
 
-require github.com/pahanini/go-grpc-bidirectional-streaming-example v0.0.0-20211027164128-cc6111af44be
+require (
+	github.com/DataDog/go-libddwaf v0.0.0-20221118102553-40bc70a9cbc3
+	github.com/pahanini/go-grpc-bidirectional-streaming-example v0.0.0-20211027164128-cc6111af44be
```

Comment:
Does this dep need to be removed?

Reply:
The  `github.com/pahanini/go-grpc-bidirectional-streaming-example` line ?
This was already there as shown by the diff. `go mod tidy` rewrote it grouped this way.